Various improvements to `install-of-alternatives.md` wiki page (#394)
PrismLauncher/prismlauncher.org: src/wiki/getting-started/install-of-alternatives.md

# Why bother?

Once upon a time, OptiFine was *the* way to make your game better, faster, and stronger through various useful features and optimizations. Things have since changed.

## Detailed explanation

> For a very long time, OptiFine was an essential mod that many enjoyed using as it offers important benefits such as performance improvements, extra quality of life features, and more. However, it also has it's downsides:

Reviewer: ```suggestion
For a very long time, OptiFine was an essential mod that many enjoyed using as it offers important benefits such as performance improvements, extra quality of life features, and more. However, it also has its downsides:
```

> - OptiFine is closed-source. This means that no one can easily inspect it's code, and it is much more difficult for developers to make their mods compatible with OptiFine - since they can't see what could be conflicting between their mod and OptiFine.

Reviewer: ```suggestion
- OptiFine is closed-source. This means that no one can easily inspect its code, and it is much more difficult for developers to make their mods compatible with OptiFine - since they can't see what could be conflicting between their mod and OptiFine.
```

- OptiFine sometimes takes a long time to update to newer Minecraft versions as there is only one developer working on the mod.

- OptiFine functions as an "all-in-one" mod (similar to a modpack). This makes it impossible for the user to disable/fully remove features that may be incompatible with other mods or that they don't need.

> In the past few years, various mods have appeared to replace these features and offer a better experience for both mod developers *and* users. Most of them are open-source, updated faster due to community contributions and support, and allow you to remove features if they're incompatible with another mod or don't suit your preference. These will usually be available on Fabric and Quilt, with some of them natively supporting (or working through compatibility layers) on Forge.

Reviewer: ```suggestion
In the past few years, various mods have been made to replace these features and offer a better experience for both mod developers *and* users. Most of them are open-source, updated faster due to community contributions and support, and allow you to remove features if they're incompatible with another mod or don't suit your preference. These will usually be available on Fabric and Quilt, with some of them natively supporting (or working through compatibility layers) on Forge.
```
"appeared to replace" could be interpreted in multiple ways

### <img src="https://cdn.modrinth.com/data/gvQqBUqZ/icon.png" height="20"><img src="https://cdn.modrinth.com/data/H8CaAYZC/icon.png" height="20"> Other Mods

> If you want to further improve your performance, please also consider installing [Lithium](https://modrinth.com/mod/lithium) and [Starlight](https://modrinth.com/mod/starlight)! Do note that Starlight is no longer needed client-side if you're playing on Minecraft 1.20 or higher, as Mojang has optimized their lighting engine since then.

Author: I used `warn` instead since I don’t think note exists unless I was doing something wrong

### <img src="https://avatars.githubusercontent.com/u/92206402?s=200&v=4" height="20"> Fabulously Optimized

If you don't want to search and install these mods manually, then try the [Fabulously Optimized](https://modrinth.com/modpack/fabulously-optimized) modpack, which supports almost all OptiFine features.

[See the installation instructions](https://fabulously-optimized.gitbook.io/modpack/readme/install-instructions#prism-launcher) for Prism Launcher.

### <img src="https://cdn-raw.modrinth.com/data/BYfVnHa7/7f8dc20fc0edd29fd95819a6f40938be0b9cadfa.png" height="20"> Simply Optimized

[Simply Optimized](https://modrinth.com/modpack/sop) is a modpack designed with just optimization in mind. SO has better out-of-the-box performance than Fabulously Optimized, but it doesn't come with the QoL mods or full OptiFine parity you would see in Fabulously Optimized, so you're expected to add any additional mods you want yourself.

### <img src="https://raw.githubusercontent.com/skywardmc/art/main/adrenaline/logo_512h.png" height="20"> Adrenaline

> [Adrenaline](https://modrinth.com/modpack/adrenaline) is a modpack which aims to improve performance as much as possible while not changing anything about the vanilla game or adding any QOL features, and remaining stable. Adrenaline supports both Fabric and Quilt and a wide range of Minecraft versions. However, like Simply Optimized, it does not come with any OptiFine replacement mods, so you'll have to manually install the features that you want.

Reviewer 2: ```suggestion
[Adrenaline](https://modrinth.com/modpack/adrenaline) is a Fabric/Quilt modpack which aims to improve performance as much as possible while not changing anything about the vanilla game, adding QOL features, or introducing instability. Like Simply Optimized, it does not come with any OptiFine replacement mods, so you'll have to manually install the features that you want.
```

### <img src="https://raw.githubusercontent.com/skywardmc/art/main/additive/logo_512h.png" height="20"> Additive

> [Additive](https://modrinth.com/modpack/additive) is a modpack similar to Fabulously Optimized which aims to support nearly all OptiFine features. It's built from Adrenaline for best performance. Additive supports both Fabric and Quilt and a wide range of Minecraft versions.

Reviewer 2: ```suggestion
[Additive](https://modrinth.com/modpack/additive) is a modpack similar to Fabulously Optimized which aims to support nearly all OptiFine features. It's based on Adrenaline for better performance, supports both Fabric and Quilt, and can be installed on a wide range of Minecraft versions.
```

## <img src="https://avatars0.githubusercontent.com/u/1390178?s=400&v=4" height="30"> Forge Mods

Unlike on Fabric/Quilt, most unofficial ports of OptiFine alternatives for Forge might have some compatibility/stability issues.

While on some modpacks/with some mods they're going to work good, with others they might be *very* unstable so be well aware of what you're doing. If you have issues, use OptiFine.

You can use [Sinytra Connector](https://modrinth.com/mod/connector) to run many Fabric mods on Forge.

**Don't report issues with those on upstream's Discord support channels!**

### <img src="https://raw.githubusercontent.com/embeddedt/embeddium/16.x/forge/src/main/resources/icon.png" height="20"> Embeddium

[Embeddium](https://modrinth.com/mod/embeddium) is a Sodium port for Forge that focuses on compatibility with other Forge mods. We generally recommend using it instead of Rubidium. Be aware of the statements above before using it.

> To install in Prism Launcher, create an instance with the **Forge** loader, and then go to the *Mods tab*, click **Download Mods**. There, search for *Embeddium*, click on it, click **Select mod for download**, and then press **OK**.

Reviewer 2: i don't think mod installation guidance is needed here. we already have the [installing mods](https://prismlauncher.org/wiki/getting-started/download-mods/) section on the wiki, which could be linked instead

edit: reading back i noticed there are other sections with this already, but imo they should be removed as well since it's pretty repetitive and duplicated information
